### PR TITLE
Table changes

### PR DESCRIPTION
**Pull request by surabhishukla**

### CODE REVIEW

File: src/Layouts/Header/Appbar.jsx [hunk 1]
```diff
@@ -56,6 +56,10 @@ navMap.set("/license/generate", {
   nav: true,
   content: true,
 });
+navMap.set("/key/modify", {
```
Comment (pinak-globallogic) [inline src/Layouts/Header/Appbar.jsx:59]: license/modify

File: src/Pages/License/Generate/GenerateProductKeyDetails.jsx [hunk 2]
```diff
@@ -55,21 +64,58 @@ const GenerateProductKeyDetails = () => {
...
-                ))}
-              </tbody>
-            </table>
+            <Grid container columnSpacing={2}>
+              <TableContainer sx={{ maxWidth: 300 }} component={Paper}>
+                <Table aria-label="simple table" size="small">
+                  <TableBody aria-label="simple table" size="small">
+                    {data.map((item, i) => (
+                      <TableRow key={i} align="left">
+                        <TableCell
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateProductKeyDetails.jsx:73]: refactor, use map for cells

File: src/Pages/License/ModifyProductKey/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import Content from "Layouts/Content/Content";
+import ProductKey from "./ProductKey";
+import LicenseDetails from "../Details";
+
+const ModifyProductKeyWrapper = () => {
+  return (
+    <Content pt={5}>
```
Comment (pinak-globallogic) [inline src/Pages/License/ModifyProductKey/index.jsx:7]: remove pt from content

File: src/Routes/index.jsx [hunk 4]
```diff
@@ -18,6 +19,7 @@ const Router = () => {
       <Route path="/license/search" element={<SearchLicenseWrapper />} />
       <Route path="/system-configuration" element={<FirstRunWrapper />} />
       <Route path={ADMIN_PANEL_PAGE_ROUTE} element={<AdminPanelWrapper />} />
+      <Route path="/key/modify" element={<ModifyProductKeyWrapper />} />
```
Comment (pinak-globallogic) [inline src/Routes/index.jsx:22]: use similar path e.g. /license/modify

File: src/Pages/License/ModifyProductKey/ProductKey.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { useSelector } from "react-redux";
+import Grid from "@mui/material/Grid";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import { Typography } from "@mui/material";
+import InputProductKey from "./InputProductKey";
+import EditProductKey from "./EditProductKey";
+
+const ProductKey = (props) => {
```
Comment (pinak-globallogic) [inline src/Pages/License/ModifyProductKey/ProductKey.jsx:10]: ModifyLicenseWizard

File: src/Pages/License/ModifyProductKey/index.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import Content from "Layouts/Content/Content";
+import ProductKey from "./ProductKey";
+import LicenseDetails from "../Details";
+
+const ModifyProductKeyWrapper = () => {
```
Comment (pinak-globallogic) [inline src/Pages/License/ModifyProductKey/index.jsx:5]: rename to ModifyLicenseWrapper.
Also, rename folder to just 'Modify'

File: .gitignore [hunk 7]
```diff
@@ -21,5 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-package-lock.json
```
[the remark this answers is not in the document]
Comment (surabhishukla) [reply .gitignore]: done

File: src/Layouts/Header/Appbar.jsx (hunk 1 above)
Comment (surabhishukla) [reply src/Layouts/Header/Appbar.jsx]: done

File: src/Pages/License/Generate/GenerateProductKeyDetails.jsx (hunk 2 above)
Comment (surabhishukla) [reply src/Pages/License/Generate/GenerateProductKeyDetails.jsx:84]: done

File: src/Pages/License/ModifyProductKey/index.jsx (hunk 3 above)
Comment (surabhishukla) [reply src/Pages/License/ModifyProductKey/index.jsx]: done

File: src/Routes/index.jsx (hunk 4 above)
Comment (surabhishukla) [reply src/Routes/index.jsx]: done

File: src/Pages/License/ModifyProductKey/ProductKey.jsx (hunk 5 above)
Comment (surabhishukla) [reply src/Pages/License/ModifyProductKey/ProductKey.jsx]: done

File: src/Pages/License/ModifyProductKey/index.jsx (hunk 6 above)
Comment (surabhishukla) [reply src/Pages/License/ModifyProductKey/index.jsx]: done

File: src/Routes/index.jsx [hunk 8]
```diff
@@ -6,7 +6,7 @@ import SearchLicenseWrapper from "Pages/License/Search";
 import AdminPanelWrapper from "Pages/Admin";
 export const LICENSE_SEARCH_PAGE_ROUTE = "/license/search";
 export const ADMIN_PANEL_PAGE_ROUTE = "/license/admin";
-import ModifyProductKeyWrapper from "../Pages/License/ModifyProductKey";
+import ModifyLicenseWrapper from "../Pages/License/Modify";
```
Comment (pinak-globallogic) [inline src/Routes/index.jsx:9]: use absolute path

File: src/Routes/index.jsx [hunk 9]
```diff
@@ -19,7 +19,7 @@ const Router = () => {
       <Route path="/license/search" element={<SearchLicenseWrapper />} />
       <Route path="/system-configuration" element={<FirstRunWrapper />} />
       <Route path={ADMIN_PANEL_PAGE_ROUTE} element={<AdminPanelWrapper />} />
-      <Route path="/key/modify" element={<ModifyProductKeyWrapper />} />
+      <Route path="/license/modify" element={<ModifyLicenseWrapper />} />
```
Comment (pinak-globallogic) [inline src/Routes/index.jsx]: Not as per the latest code. You have removed the upcoming changes during conflicts. Revert

File: src/Layouts/Header/Appbar.jsx [hunk 10]
```diff
@@ -56,7 +56,7 @@ navMap.set("/license/generate", {
   nav: true,
   content: true,
 });
-navMap.set("/key/modify", {
+navMap.set("/license/modify", {
```
Comment (pinak-globallogic) [inline src/Layouts/Header/Appbar.jsx]: Not as per the latest changes, resolve conflicts carefully

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/Pages/License/Modify/ModifyLicenseWizard.jsx [hunk 11]
```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { useSelector } from "react-redux";
+import Grid from "@mui/material/Grid";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import { Typography } from "@mui/material";
+import InputProductKey from "./InputProductKey";
+import EditProductKey from "./EditProductKey";
+
+const ProductKey = (props) => {
```
Comment (pinak-globallogic) [inline src/Pages/License/Modify/ModifyLicenseWizard.jsx:10]: Change the component name and file name to  ModifyProductKeyWizard

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

**Review by pinak-globallogic — APPROVED** [review]

**Pull request merged**